Tests/CustomFontTests.swift

```diff
@@ -49,17 +49,17 @@ class CustomFontTests: XCTestCase {
     //MARK: - Dynamic Type - Deprecated in iOS 11
     
     func testOfStyle() {
-        let font = Futura.medium.of(style: .title3)
+        let font = Futura.medium.of(textStyle: .title3)
```

Comment:
If we are removing support for < iOS 12, these tests can be deleted along with the deprecated `of(style:)` function.